servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -207,28 +208,18 @@ public LogResponse getCommitLog(
     final Script script;
     try {
       script =
-          SCRIPT_HOST.getOrCreateScript(
-              queryExpression, COMMIT_LOG_DECLARATIONS, Collections.emptyList());
+          SCRIPT_HOST
```

Comment:
Can you use the `buildScript` method for entries filtering as well, so it doesn't raise a deprecation-warning.

Reply:
updated